package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-grants-frontend",
-  "version": "1.73.2",
+  "version": "1.74.2",
```

Comment:
This should be 1.74.0 (aways reset the PATCH to 0 when you increment the MINOR)

Reply:
ok got it read an article about semantic someone mentioned not to reset (confusing)